Dockerfile

```diff
@@ -79,6 +79,16 @@ RUN /bin/bash -c 'if [ "${USE_DOT_NET}" = "yes" ] ; then \
   mv /root/.dotnet/* /usr/local/dotnet; \
 fi'
 
+# if kubectl is set then install kubectl
+ARG USE_KUBECTL
+RUN /bin/bash -c 'if [ "${USE_KUBECTL}" = "yes" ] ; then \
```

Comment:
Use KUBECTL_VERSION here. If version is `latest`, then use curl to get latest. Otherwise use the configured version. Default to latest in docker-compose.yml.